tensorflow/core/kernels/data/experimental/csv_dataset_op.cc

```diff
@@ -126,11 +141,29 @@ class CSVDatasetOp : public DatasetOpKernel {
         ctx, select_cols.empty() || select_cols.front() >= 0,
         errors::InvalidArgument("select_cols should be non-negative indices"));
 
-    *output = new Dataset(ctx, std::move(filenames), header,
-                          std::move(compression_type), zlib_compression_options,
-                          output_types_, output_shapes_,
-                          std::move(record_defaults), std::move(select_cols),
-                          use_quote_delim, delim[0], std::move(na_value));
+    std::vector<int64> exclude_cols;
+    exclude_cols.reserve(exclude_cols_tensor->NumElements());
+    for (int i = 0; i < exclude_cols_tensor->NumElements(); ++i) {
+      exclude_cols.push_back(exclude_cols_tensor->flat<int64>()(i));
+    }
```

Comment:
You'll need to remove this too (this clashes with the above suggestion you committed)